aiidalab_widgets_base/structures.py

```diff
@@ -688,9 +694,13 @@ def _pybel_opt(self, smiles, steps):
         pbmol.localopt(forcefield="mmff94", steps=100)
 
         f_f = pb._forcefields["uff"]  # pylint: disable=protected-access
-        f_f.Setup(pbmol.OBMol)
+        if not f_f.Setup(pbmol.OBMol):
+            raise ValueError("OpenBabel: Missing UFF parameters")
```

Comment:
Are we sure we need to raise an exception here? We might just want to put the message into `self.output`?

Reply:
The same is true for the similar things down below.

Reply:
So I am actually catching `ValueError` on line 754 when calling this function (as it was kind of done in the original implementation), and appending the error to self.output when the exception is caught. Not sure what is preferable here.